services/server/src/server/services/compiler/lambda/SolcLambda.ts

```diff
@@ -38,10 +39,12 @@ export class SolcLambda implements ISolidityCompiler {
     solcJsonInput: JsonInput,
     forceEmscripten: boolean = false
   ): Promise<CompilerOutput> {
-    const response = await this.invokeLambdaFunction(
-      JSON.stringify({ version, solcJsonInput, forceEmscripten })
-    );
-    return this.parseCompilerOutput(response);
+    const param = JSON.stringify({ version, solcJsonInput, forceEmscripten });
+    logger.debug("Invoking Lambda function", { param });
```

Comment:
It does not make a difference how it's logged. It will be stringified in the end. `param` needed to be stringified anyway for the `invokeLambdaFunction`